larqdocs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
In the future we should probably write a proper mkdocs plugin for this. For now, this will work